tpl/bootstrap5.php

```diff
@@ -477,11 +477,11 @@
 					<div id="placeholder" class="col-md-12 hidden"><?php echo I18n::_('+++ no document text +++'); ?></div>
 					<div id="attachmentPreview" class="col-md-12 text-center hidden"></div>
 					<h6 id="copyShortcutHint" class="col-md-12"><small id="copyShortcutHintText"></small></h6>
+					<button type="button" id="prettyMessageCopyBtn" class="text-secondary opacity-05-1-hover col-md-12 hidden">
```

Comment:
Similar to the bootstrap template, inserting the button here will display it in the top right corner, but over the top navigation. Again this may simply require tweaking the CSS style for that button ID found in css/bootstrap5/privatebin.css.